edx_notifications/server/web/static/edx_notifications/js/models/counter_icon_model.js

```diff
@@ -0,0 +1,11 @@
+var CounterIconModel = Backbone.Model.extend({
+    /* model for the Notification Icon which will display a counter of unread messages */
+
+    defaults: {
+        /* start with unknown number of unread notifications */
+        'count': null
+    },
+
+    /* REST API endpoint to use to get authenticated user's notification count */
+    url: "/api/edx_notifications/v1/consumer/notifications/count?unread=True&read=False"
```

Comment:
Yes, that's one way that we do it. The other way is to pass it as a parameter to the root view as long as the view is constructed by JS output from a Mako template.